BitwardenShared/UI/Platform/Application/Extensions/View+Toolbar.swift

```diff
@@ -77,6 +77,21 @@ extension View {
         .frame(minHeight: 44)
     }
 
+    /// Returns a `Button` that displays an image for use in a toolbar.
+    ///
+    /// - Parameters:
+    ///   - label: The label associated with the image, used as an accessibility label.
+    ///   - action: The action to perform when the button is tapped.
+    /// - Returns: A `Button` for displaying an image in a toolbar.
+    ///
+    func toolbarButton(_ label: String, action: @escaping () -> Void) -> some View {
+        Button(label, action: action)
+            // Ideally we would set both `minHeight` and `minWidth` to 44. Setting `minWidth` causes
+            // padding to be applied equally on both sides of the image. This results in extra padding
+            // along the margin though.
+            .frame(minHeight: 44)
+    }
```

Comment:
❓ This is almost the same as the function below this, just without an async action. Could we use that instead?

https://github.com/bitwarden/ios/blob/3923b990c67bc5defe8f97f8eff0b2d0730c314c/BitwardenShared/UI/Platform/Application/Extensions/View%2BToolbar.swift#L102-L109

Reply:
yeah, good call.  It felt wrong at first to use the `AsyncButton` for makes sense the more i think on it.

Reply:
Just for tracking purposes, ended up keeping both functions because updating the tests to make them async even though they weren't just didnt feel right.  Discussed making one function with 2 optional closures (action/asyncAction) but felt keeping both functions would be easier to understand and use.